addons/mail/static/src/model/store.js

```diff
@@ -101,6 +122,9 @@ export class Store extends Record {
                     /** @type {[Record, Map<string, true>]} */
                     const [record, map] = FU_QUEUE.entries().next().value;
                     FU_QUEUE.delete(record);
+                    if (RHD_QUEUE.has(record) || RD_QUEUE.has(record)) {
+                        return;
+                    }
```

Comment:
Some compute were called while record was flagged for deletion (or logically deleted but not "hardly" deleted, the H in RHD). This skips handling these compute.